js/animations.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function () {
+    $('a.nav__link').click(function() {
+
+            event.preventDefault();
+            $('body').animate({
```

Comment:
The smooth scroll works for me only after that line is modified to:
`$('html, body').animate({`